--- Projects/Payment/Payment.xcodeproj/project.pbxproj	2021-07-04 10:22:59.000000000 +0200
+++ Projects/Payment/Payment.xcodeproj/project 2.pbxproj	2021-07-04 10:21:27.000000000 +0200
@@ -32,7 +32,7 @@
 		34EED3EF358017396F65A79D /* MyPayment.swift in Sources */ = {isa = PBXBuildFile; fileRef = DDBC6D9ACF5A036A8FEABFE9 /* MyPayment.swift */; };
 		381495239262AE27B0BDF570 /* Testing.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 661B5529AF1881FA2164DA02 /* Testing.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		3896EA5C67EFB96181C6BBC6 /* AdyenPayOut.swift in Sources */ = {isa = PBXBuildFile; fileRef = A07B88F89A4130881F71FEEA /* AdyenPayOut.swift */; };
-		3E4B3F8A2B5ECF2A95256BA2 /* NonMacDependencies.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6A52FEF12C151697F1248FCC /* NonMacDependencies.framework */; };
+		3E4B3F8A2B5ECF2A95256BA2 /* NonMacDependencies.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6A52FEF12C151697F1248FCC /* NonMacDependencies.framework */; platformFilter = ios; };
 		3FF8D41BA35FC5B51BAE6DD3 /* MyPayment.graphql.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6A419AE3DF5EB7934FCCFE62 /* MyPayment.graphql.swift */; };
 		44D6F3D1A2EDB877A66777D9 /* CoreDependencies.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5B39E627376ADC2E74E7FF50 /* CoreDependencies.framework */; };
 		476BE9812380D4B4715EA756 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 60717183B676C86211A4801F /* AppDelegate.swift */; };